MTFO/Patches/Patch_DiscordManager.cs

```diff
@@ -0,0 +1,23 @@
+using MTFO.Utilities;
+using HarmonyLib;
+
+namespace MTFO.Patches
+{
+    [HarmonyPatch(typeof(DiscordManager), nameof(DiscordManager.UpdateDiscordDetails))]
+    static class Patch_DiscordManager
+    {
+        [HarmonyPrefix]
+        private static bool Pre_UpdateDiscordDetails(eDiscordDetailsDisplay details)
+        {
+            if (details == null) return false;
```

Comment:
```suggestion
            if (details == null) return true;
```

Isn't eDiscordDetailsDisplay an enum? How can this be null? Assuming the naming convention is misused (classic 10cc), let the original method handle this.